### ISSUE TITLE
1.0.0: cannot disable multiline-expression-wrapping

### ISSUE THREAD

**Issue by hoc081098** [pinterest/ktlint#2288]
Hi, I want to disable `multiline-expression-wrapping` rule, but it seems the disabling has no effect.
Please help me, thank you so much 🙏 

```
ktlint_standard_multiline-expression-wrapping=disabled
ktlint_standard_statement-wrapping=disabled
ktlint_standard_discouraged-comment-location=disabled
ktlint_standard_if-else-wrapping=disabled
```

![image](https://github.com/pinterest/ktlint/assets/36917223/2e4c75ba-c289-4ed0-9cdf-6a920e18a437)

**Comment by paul-dingemans** [comment]
It might also have been caused by function-signature rule depending on your settings. When you run Ktlint without format, the rule id is listed. See docs how to change behavior of that rule.

**Comment by SimonMarquis** [comment]
I can report the same behavior here, even with `ktlint_code_style = intellij_idea` which should have the rule disabled by default:

```kotlin
    private var colorSeeds =
        intArrayOf(
            Color.parseColor("#000000"),
            Color.parseColor("#FF5252"),
            Color.parseColor("#FFEB3B"),
            Color.parseColor("#00C853"),
            Color.parseColor("#00B0FF"),
            Color.parseColor("#D500F9"),
            Color.parseColor("#8D6E63"),
        )
```

```kotlin
    override fun onPostCreate(savedInstanceState: Bundle?) {
        super.onPostCreate(savedInstanceState)
        videoRecorder =
            VideoRecorder(this, arSceneView) { isRecording ->
                ...
            }
```

```kotlin
        thumbRadius =
            (barHeight / 2).toFloat().let {
                if (it < minThumbRadius) minThumbRadius else it
            }
```

```kotlin
val screenshotHandler =
    HandlerThread("screenshot")
        .also { it.start() }
        .let { Handler(it.looper) }
```

**Comment by paul-dingemans** [comment]
> I can report the same behavior here, even with `ktlint_code_style = intellij_idea` which should have the rule disabled by default (running KtLint through spotless Gradle plugin):

Please try to reproduce the problem with Ktlint CLI first. I believe this is a bug in the spotless integration with ktlint.

**Comment by SimonMarquis** [comment]
I confirm this behavior does not happen when running the CLI commands.
